PIL/Image.py

```diff
@@ -1209,6 +1278,7 @@ def getextrema(self):
         return self.im.getextrema()
 
     def getim(self):
+        # type: () -> Any
```

Comment:
Ohhh, okay, that makes sense! I have no idea how to annotate that then. May as well be `Any`.